### ISSUE TITLE
these plugin not working in ios real devices

### ISSUE THREAD

**Issue by ayushbafna24** [ibm-bluemix-mobile-services/bms-clientsdk-cordova-plugin-push#94]
These plugin has completely failed When I am running on a ios real devices.I have tried everything to run my app by using these plugin.I have used xcode 9.It shows me a error "dylib" still.I have also raised a bluemix support ticket.No one can support me for these issue.

**Comment by AnanthaKrish** [comment]
@ayushbafna24 Please share the logs or Errors

**Comment by ayushbafna24** [comment]
![screen shot 2018-02-27 at 7 00 47 pm](https://user-images.githubusercontent.com/12123191/36731484-d21fea4c-1bc2-11e8-9599-8d895f63e004.png)

**Comment by ayushbafna24** [comment]
Logs:-

dyld: Library not loaded: @rpath/BMSAnalytics.framework/BMSAnalytics
  Referenced from: /var/containers/Bundle/Application/466E9986-61F2-4347-87F0-5CB614CF841B/ePricer BP.app/ePricer BP
  Reason: no suitable image found.  Did find:
	/private/var/containers/Bundle/Application/466E9986-61F2-4347-87F0-5CB614CF841B/ePricer BP.app/Frameworks/BMSAnalytics.framework/BMSAnalytics: code signing blocked mmap() of '/private/var/containers/Bundle/Application/466E9986-61F2-4347-87F0-5CB614CF841B/ePricer BP.app/Frameworks/BMSAnalytics.framework/BMSAnalytics'
(lldb)

**Comment by ayushbafna24** [comment]
If I have removed these plugin on my project.My project works fine on a real device of iphone and If I have used these plugin.I am getting an error everytime.My issues has not solved for more than one month.

**Comment by AnanthaKrish** [comment]
@ayushbafna24 please open your keychain app , then open the iPhone developer certificate, expand the trust option and change the ‘when using this certificate’ field to ‘Use System Defaults’

**Comment by ayushbafna24** [comment]
I have tried with your solution.These solution has not worked for me.Still I have a major issues for that.

**Comment by ayushbafna24** [comment]
Gentle Reminder.Solve my issues When I am running on a ios real devices.

**Comment by AnanthaKrish** [comment]
@ayushbafna24  Are you opening the `.xcworkspace ` or `xcodeproj ` ??

**Comment by ayushbafna24** [comment]
Yes I have opened and tried with your solution.If you want I have sharing you a screenshot also.

**Comment by AnanthaKrish** [comment]
@ayushbafna24 

1. Go to iOS folder , do a pod install
2. open `yourproject. xcworkspace ` , and build.  (don't open `yourproject. xcodeproj` )

This should work for you .

**Comment by ayushbafna24** [comment]
Please arrange a sametime meeting with me for your suitable office hours.I have already tried with these pod installation command.These has not helped to me.Please arrange a live session with me.I have show you everything.If you do this.These is good for me.

**Comment by ayushbafna24** [comment]
Gentle Reminder I have already tried your solution.Still I have faced these issue
![screen shot 2018-03-13 at 10 48 00 am](https://user-images.githubusercontent.com/12123191/37323905-5dfbf87e-267e-11e8-81f5-2ee32e49f4a6.png)

**Comment by ayushbafna24** [comment]
Gentle  Reminder

**Comment by AnanthaKrish** [comment]
@ayushbafna24 Can you raise a Support ticket please ?

**Comment by ayushbafna24** [comment]
I have already raised a support ticket.If you want a ticket number.I will share you a screenshot for these.Solve my issue soon as possible.I have faced these issue for more than a month.

**Comment by ayushbafna24** [comment]
Reminder ...Waiting for a Kind Response

**Comment by ayushbafna24** [comment]
Gentle Reminder....
